src/python/gudhi/dtm_rips_complex.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Yuichi Ike, Raphaël Tinarrage
+#
+# Copyright (C) 2020 Inria, Copyright (C) 2020 FUjitsu Laboratories Ltd.
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+
+from gudhi.weighted_rips_complex import WeightedRipsComplex
+from gudhi.point_cloud.dtm import DistanceToMeasure
+from scipy.spatial.distance import cdist
+
+class DtmRipsComplex(WeightedRipsComplex):
+    """
+    Class to generate a DTM Rips complex from a distance matrix or a point set, 
+    in the way described in :cite:`dtmfiltrations`.
+    Remark that all the filtration values are doubled compared to the definition in the paper 
+    for the consistency with RipsComplex.
+    """
+    def __init__(self, 
+                 points=None, 
+                 distance_matrix=None, 
+                 k=1, 
+                 q=2,
+                 max_filtration=float('inf')):
+        """
+        Args:
+            points (Sequence[Sequence[float]]): list of points.
+            distance_matrix (ndarray): full distance matrix.
```

Comment:
We *could* use a name other than `distance_matrix` (`full_distance_matrix`? `distances`?) to reflect the fact that it has stricter expectations than `distance_matrix` in `RipsComplex`, but I don't think that's necessary, `distance_matrix` looks good enough to me.
(just posting this in case someone disagrees)

Reply:
`distance_matrix` looks good enough also to me.